src/Backups/BackupSettings.h

```diff
@@ -38,6 +38,9 @@ struct BackupSettings
     /// Whether the BACKUP will omit similar files (within one backup only).
     bool deduplicate_files = true;
 
+    /// Whether native copy is allowed (optimization for cloud storages, that sometimes could have bugs)
+    bool native_copy = true;
```

Comment:
It's going to allow native copy for S3, so I suggest renaming it to `allow_s3_native_copy`.

Reply:
I think it is good to keep it generic, maybe there will be more implementations for native copy for some other cloud storages in future, but if you insist I could rename.